pages/collections/la-rebellion/index.vue

```diff
@@ -0,0 +1,81 @@
+<script setup>
+
+// HELPERS
+import _get from 'lodash/get'
+
+// GQL
+import FTVACollectionStory from '../gql/queries/FTVACollectionStory.gql'
+
+const { $graphql } = useNuxtApp()
+
+const route = useRoute()
+console.log('route path: ', route.path)
+
+// routes this template/page supports:
+const routeNameToSlugMap = {
+  '/collections/la-rebellion': 'la-rebellion-2',
```

Comment:
I fixed the slug in craft for `in the life` and `LA Rebellion`, so use `in-the-life` and `la-rebellion` for story format entry>

https://test-craft.library.ucla.edu/admin/entries/ftvaCollectionStory/3990584-in-the-life
https://test-craft.library.ucla.edu/admin/entries/ftvaCollectionStory/3974103-la-rebellion